x-pack/plugins/security_solution/public/detections/components/rules/step_define_rule/index.tsx

```diff
@@ -981,14 +983,7 @@ const StepDefineRuleComponent: FC<StepDefineRuleProps> = ({
 
           <EuiSpacer size="m" />
           <RuleTypeEuiFormRow $isVisible={isAlertSuppressionEnabled && isThresholdRule} fullWidth>
-            <EuiToolTip
-              content={
-                !isAlertSuppressionLicenseValid
-                  ? i18n.ENABLE_THRESHOLD_SUPPRESSION_LICENSE_WARNING
-                  : null
-              }
-              position="right"
-            >
+            <EuiToolTip content={alertSuppressionUpsellingMessage} position="right">
```

Comment:
Should we check if the upselling message exists before rendering the tooltip?

Reply:
if `content` property is `null`, tooltip won't be rendered. So, no need to do check if message does not exists

Reply:
Yeah, but that depends on the EuiTooltip internal implementation. If we don't want to show anything it's better to not render the tooltip at all. Like it's done on the other upselling message [here](https://github.com/elastic/kibana/pull/172555/files#diff-93d55fa79ac01543bdc8f90f67c92a681b6ef7428f4ffe86e549c0b1c883e3f6R24-R28).

That's just a NIT anyway. It looks good overall 👍

Reply:
There is small difference in behaviour of the both components

- in this example, if no upselling message available, we show children component `CommonUseField`, which is enabled when Platinum license available and disabled with upselling message when license is not Platinum
- in example from [alert_suppression_technical_preview_badge.tsx](https://github.com/elastic/kibana/pull/172555/files#diff-93d55fa79ac01543bdc8f90f67c92a681b6ef7428f4ffe86e549c0b1c883e3f6R24-R28), when no upselling message available we do not render any component at all. Thus check is needed

Reply:
Oh yes, you are right, I did not notice this one has children. everything makes sense now, my bad 👍